index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Abel portfolio</title>
+  </head>
+  <body>
+    <!-- Navbar -->
+    <nav class="nav">
+      <logo>
+        <p class="logo">Abel Tsegaye</p>
+      </logo>
+      <img class="mobileMenu" src="images\IconMenu.png" alt="Menu" />
+    </nav>
```

Comment:
- [ ] Kindly make sure to wrap the navigation container into the `header` tag. Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.